README.md

```diff
@@ -185,11 +185,10 @@ To publish locally, you'll need to do the following...
 ####Sign In
 
 To publish locally, you'll need an account.
-* Go to [http://localhost:3000/account](http://localhost:3000/account)
-* Click ``Join Webmaker`` and complete the process, you can use a fake email
-* When you've created your account, click ``Set permanent password instead``
-  * This lets you authenticate your account without needing email
-* Go back to Thimble and Log In with your new account
+* Go to [http://localhost:3000](http://localhost:3000)
```

Comment:
We should tell them to do it through Thimble, as it'll send them to the ```id``` server with the styled login page just like on staging or production. It'll also pass the correct parameters and return back to Thimble afterwards. I think this documentation could certainly use some updating though.